src/helpers/cache.ts

```diff
@@ -0,0 +1,28 @@
+import LRUCache from 'lru-cache';
+
+class Cache extends LRUCache<string, string> {
+  // OVERRIDING THIS METHODS TO ENFORCE STORING OF STRING VALUES
```

Comment:
Great documentation practice, would suggest looking up for an even better vscode extension to better document the functions and methods. One of them would be **(Document this).**